cookbook/configuration/override_dir_structure.rst

```diff
@@ -176,8 +176,7 @@ The change in the ``composer.json`` will look like this:
         ...
     }
 
-In ``app/autoload.php``, you need to modify the path leading to the
-``vendor/autoload.php`` file::
+In ``app/autoload.php``, you need to modify the path to the ``autoload.php`` file::
```

Comment:
I like your proposal ... but in the future I'd like to review all the cookbook articles to be more concise and straight-to-the-point. For example, this phrase could be reworded as:

```
Then, update the path to the ``autoload.php`` file in ``app/autoload.php``::
```